pyuntl/untl_structure.py

```diff
@@ -26,7 +25,7 @@ def create_untl_xml_subelement(parent, element, prefix=''):
         subelement.text = element.content
     if element.qualifier is not None:
         subelement.attrib["qualifier"] = element.qualifier
-    if element.children > 0:
+    if element.children:
         for child in element.children:
             SubElement(subelement, prefix + child.tag).text = child.content
     else:
```

Comment:
My understanding is that there is a situation where we could hit the `else` here and _not_ duplicate work. Specifically, if the UNTLElement has _neither_ content nor children, then line 25 would not have been run before entering the `else` clause.

Reply:
Yeah, I think my thought (though it was a while back now) was that if the `element.content` is `None`, you don't need to set `subelement.text` because its value should already `None`.

Reply:
Yeah that makes sense. So I guess we could just get rid of the `else`